### DISCUSSION TITLE
Configurable Title / Subtitle separator?

### DISCUSSION THREAD

**Discussion by tillmanj** [aidenlx/obsidian-zotero#259] [Ideas]
Given that Obsidian cannot have a colon in the title and given that the colon is the single most common separator character between title and subtitle in books, would it be possible to make the substitution character a configurable option? Some might want to use a dash of some sort, some prefer the vertical ellipsis (︙).

Or perhaps this is a configuration option hiding deep inside Obsidian or Zotero that I just need to be pointed towards.

**Comment by axbax** [comment]
Hi, I experience an issue with this. I want to have the title in the 'aliases' property in Obsidian. This will not accept titles with a colon (:), so it when I try to create a literature note with a colon in the title it tells me that it 'Failed to renderYaml'. Thanks for any help to resolve this!

**Comment by aidenlx** [comment]
In v2 the filename is a template you own, so the substitution character is your choice rather than a fixed one:

```liquid
{{ zt.title | replace: ": ", " — " }}
```

ZotLit still falls back to `_` for characters Obsidian genuinely forbids, but if you rewrite the colon yourself it never reaches that fallback.

Separately, the *Failed to render YAML* on an `aliases` field containing a colon, reported in the comment above, is fixed — v2 serializes frontmatter values properly and quotes them where needed, so a title with a colon saves fine.

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
